src/current/_includes/molt/molt-install.md

```diff
@@ -12,7 +12,22 @@ To install MOLT, download the binary that matches your architecture and source d
 The following binaries are included:
 
 - `molt`
-- `replicator`. To use replication features, `replicator` **must** be located either in the same directory as `molt` or in a directory beneath `molt`.
+- `replicator`
+
+Both `molt` and `replicator` **must** be in your current working directory. To use replication features, `replicator` must be located either in the same directory as `molt` or in a directory directly beneath `molt`. For example, either of the following would be valid:
```

Comment:
Do you think it's worth bolding "working directory" since this is a bit of a gotcha.